codechecker_lib/analyzers/result_handler_plist_to_file.py

```diff
@@ -0,0 +1,76 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+
+import ntpath
+import os
+import shutil
+from abc import ABCMeta
+
+from codechecker_lib import plist_parser
+from codechecker_lib.logger import LoggerFactory
+from codechecker_lib.analyzers.result_handler_base import ResultHandler
+
+LOG = LoggerFactory.get_new_logger('PLIST TO FILE')
+
+
+class PlistToFile(ResultHandler):
+    """
+    Result handler for copying a plist file to a different location.
+    """
+
+    __metaclass__ = ABCMeta
+
+    def __init__(self, buildaction, workspace, export_plist_path):
+        super(PlistToFile, self).__init__(buildaction, workspace)
+        self.__folder = export_plist_path
+
+    @property
+    def print_steps(self):
```

Comment:
Is this variable used _anywhere_ in the codebase at all? I don't seem to find any actual usage apart from propery accessors.